tests/Unit/HtmlProcessor/AbstractHtmlProcessorTest.php

```diff
@@ -224,9 +224,9 @@ public function provideHtmlWithOptionalTagsOmitted(): array
             // broken: OPTGROUP end tag omission with OPTGROUP element following
             'OPTGROUP end tag omission at end of list' => [
                 '<select><optgroup><option> 1 </option><option> 2 </option></optgroup>'
-                    . '<optgroup><option> A </option><option> B </option></select>',
+                . '<optgroup><option> A </option><option> B </option></select>',
```

Comment:
I think it's more readable with the original indentation, making it clearer where each array element definition begins.

Reply:
Done.